algebra-core/src/lib.rs

```diff
@@ -6,13 +6,13 @@
 #![deny(unused_comparisons, bare_trait_objects, const_err, unused_must_use)]
 #![deny(unused_mut, unused_unsafe, private_in_public)]
 #![cfg_attr(
-    all(
+    not(all(
```

Comment:
Hmm why do we want to negate here? `unsafe` is only required atm when all of these conditions are satisfied, right?